### ISSUE TITLE
Force close  in Fedora 28 (Workaround)

### ISSUE THREAD

**Issue by sefernet** [autokey/autokey#191]
## Classification:
Bug

## Reproducibility:
Always

## Summary
Force close in Fedora 28, presumible due missing dependency "gtksourceview3"

## Steps to Reproduce (if applicable)
- Install "autokey-gtk" from Fedora official repositories
- Run "autokey-gtk"
- Force close

## Actual Results
Error from console:
$ autokey-gtk 
```
Traceback (most recent call last):
  File "/bin/autokey-gtk", line 11, in <module>
    load_entry_point('autokey==0.94.0', 'console_scripts', 'autokey-gtk')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 476, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2700, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2318, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.6/site-packages/autokey/gtkui/__main__.py", line 4, in <module>
    from autokey.gtkapp import Application
  File "/usr/lib/python3.6/site-packages/autokey/gtkapp.py", line 33, in <module>
    from autokey.gtkui.configwindow import ConfigWindow
  File "/usr/lib/python3.6/site-packages/autokey/gtkui/configwindow.py", line 22, in <module>
    require_version('GtkSource', '3.0')
  File "/usr/lib64/python3.6/site-packages/gi/__init__.py", line 134, in require_version
    (namespace, version))
ValueError: Namespace GtkSource not available for version 3.0
```
## Version
0.94.0

## Distro:
Fedora 28 

## Workaround:
In Fedora, install "gtksourceview3"
$  dnf install gtksourceview3

**Comment by josephj11** [comment]
TL;DR: sounds like a Fedora packaging issue - to be resolved by the Fedora packagers.

Thanks for reporting the workaround!

If that fixes it, doesn't that indicate that the problem is a packaging error in Fedora?

It's really great they are supporting a newer version now. (They were still packaging 0.90.4 - six years old - until very recently.) Maybe their package needs tweaking.

We have had reports from several users who have successfully installed AutoKey on Fedora using pip3, but I think they were on 26 or 27 and they had to wrangle the dependencies themselves.

FYI, the current version (available from pip3) is 0.95.3 last I checked. 0.94.0 is still fairly recent.

**Comment by luziferius** [comment]
Just for reference: These are the packaged versions for each Fedora release: https://apps.fedoraproject.org/packages/autokey/overview/
